courses/intro-to-ada/chapters/strongly_typed_language.rst

```diff
@@ -658,3 +658,54 @@ an exception will be raised.
        --         ^ Wrong value for the subtype
        --           Compiles, but exception at runtime
     end Greet;
+
+Type aliases
+------------
+
+Previously, we've seen that we can create new types by declaring
+:ada:`type Miles is new Float`. We could also create type aliases, which
+generate alternative names |mdash| *aliases* |mdash| for known types. We achieve
+this by using subtypes. In this case, however, we don't get the benefits of
```

Comment:
I would say "we don't get all of the benefits" - without "all of", implies (to me) no benefits - we still get the benefits from the parent